frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/FastForwardMerge.java

```diff
@@ -25,34 +25,37 @@ private static void mergeCurrentBranch(Project project,
 
   private static void mergeNonCurrentBranch(Project project,
       GitRepository gitRepository,
-      MergeProps mergeProps) {
+      MergeProps mergeProps, String fetchNotification) {
     val stayingFullName = mergeProps.getStayingBranch().getFullName();
     val movingFullName = mergeProps.getMovingBranch().getFullName();
     val refspecFromChildToParent = createRefspec(stayingFullName, movingFullName, /* allowNonFastForward */ false);
     val stayingName = mergeProps.getStayingBranch().getName();
     val movingName = mergeProps.getMovingBranch().getName();
+    val successFfMergeNotification = fetchNotification
```

Comment:
Two-letter [acronyms & initialisms](https://www.rd.com/article/acronym-vs-abbreviation-whats-the-difference/) are fully capitalized in camel case as per Java conventions:
```suggestion
    val successFFMergeNotification = fetchNotification
```

Only up from three letters is upper case used (`Tls` and not `TLS`)